nautobot/ipam/api/serializers.py

```diff
@@ -235,8 +240,19 @@ class Meta:
             "ip_version": {"read_only": True},
             "mask_length": {"read_only": True},
             "nat_outside_list": {"read_only": True},
+            "parent": {"required": False},
         }
 
+    def validate(self, data):
+        namespace = data.get("namespace", None)
+        parent = data.get("parent", None)
+
+        # Only assert namespace/parent on create.
+        if self.instance is None and not any([namespace, parent]):
+            raise ValidationError({"__all__": "One of parent or namespace must be provided"})
```

Comment:
This should be documented in the "API Behavior Changes" section of `upgrading-from-nautobot-v1.md`.

Reply:
In fact I'm a bit concerned that the term `namespace` doesn't appear even once in that document presently... 😭

Reply:
Now is the time!